### DISCUSSION TITLE
On demand source stops afer 10 seconds

### DISCUSSION THREAD

**Discussion by tompointexe** [bluenviron/mediamtx#947] [Questions]
Hey, i'm setuping a on demand source on a remote server as a "tiny cdn" for myself, and i'm running into an "issue" were the streams stops after 10 seconds

**Comment by rapi3** [comment]
Try v0.18.2 or lower

**Reply by tompointexe** [reply]
Yup, no issues on V0.18.2

[a comment by an automated account is not shown]
